hq_superset/views.py

```diff
@@ -63,6 +65,8 @@ def _ucr_id_from_pk(self, datasource_pk):
             return None
 
     @expose("/update/<datasource_id>", methods=["GET"])
+    @has_access
+    @permission_name("write")
```

Comment:
these checks were missing earlier?

Reply:
Yes, but earlier we didn't need them. Now we need to set them to explicitly disable a read-only user from performing this action.